lib/menu.rb

```diff
@@ -0,0 +1,27 @@
+class Menu
+  attr_reader :menu
+
+  THE_MENU = {'Chicken Tacos' => 6,
+    'Burrito' => 8,
+    'Enchiladas' => 7,
+    'Quesadillas' => 8,
+    'Nachos' => 5,
+    'Sangria' => 3,
+    'Water' => 2
+  }
```

Comment:
So much smarter to just create a hash than trying to actually build a menu containing dish objects :D